service_mesh/v2x/ossm-security.adoc

```diff
@@ -7,6 +7,15 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource].
```

Comment:
@rcernich Jaime's issue, https://issues.redhat.com/browse/OSSMDOC-151, says that there are implications for Kube NetworkPolicy. But, your security guide https://docs.google.com/document/d/1YsyoZn9gxRP8P3Vwm5-SGxQ1XV9JuGApzgDADiK86KQ/edit#heading=h.a5017lgbe5a6, says that its not required once you create the SMMR. 

Is that the end of the story, or are there other issues if someone already has a NetworkPolicy object?

Reply:
I think the issue with NetworkPolicy is that once the namespace joins the mesh, access to services/pods in that namespace will not be accessible to callers outside the mesh.